v2/http-middleware/ratelimit/ratelimit.go

```diff
@@ -52,7 +51,7 @@ func CreateLimiter(s Store) Limiter {
 // If you give multiple configs for 1 endpoint. The most restrictive one will apply
 // The algorithm is inspired from: https://redislabs.com/redis-best-practices/basic-rate-limiting/
 func (s *Limiter) Configure(config EndpointConfig) {
```

Comment:
Here we could skip the EndointConfig struct and just use two arguments? Maybe add a type for the `func(*http.Request) ([]Config, error){}`

Reply:
and use `time.Minute `to make it more clear what we are doing.

Reply:
Where should I use time.Minute? the requests per minute is not minutes?